lib/constants.js

```diff
@@ -27,8 +23,12 @@ var eventArgs = exports.eventArgs = {
   'throw': ['error']
 };
 
-
-var errMsgs = exports.errMsgs = {
+var errMsgs = {
   'err_plugin_type': 'Plugin type {{type}} does not exist!',
   'err_token': 'No token found!'
 };
+
+exports = {
```

Comment:
I propose using Babel! Actually I wonder how easy it is to integrate with
nodejs
On Oct 30, 2015 15:33, "Elvin Yung" <notifications@github.com> wrote:

> In lib/constants.js
> <https://github.com/elvinyung/botstrap/pull/1#discussion_r43543385>:
>
> >    'err_plugin_type': 'Plugin type {{type}} does not exist!',
> >    'err_token': 'No token found!'
> >  };
> > +
> > +exports = {
>
> This should be module.exports
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elvinyung/botstrap/pull/1/files#r43543385>.
>